### PR TITLE
Add stereo_reconstructor messages

### PR DESCRIPTION
**Pull request by WilliamLeeMoor**
Updates the placeholder stereo_reconstructor_msgs.h to include the skeleton messages for the stereo_reconstructor app 

## Description
Just basic skeleton app messages; nothing unique to stereo_reconstructor yet

## 
Related issue (Perception - Skeleton App): https://github.com/PlanetaryRobotics/moonranger-rover-flight/issues/48

## How has this been tested?
Compiled alongside scratch/stereo_reconstructor branch of the main repo. Builds and all unit tests pass.

## Screenshots (if appropriate):

```Running tests...
Test project /home/moonranger/moonranger-rover-flight/build
      Start  1: CS
 1/15 Test  #1: CS ...............................   Passed    2.40 sec
      Start  2: LC
 2/15 Test  #2: LC ...............................   Passed    2.22 sec
      Start  3: SC
 3/15 Test  #3: SC ...............................   Passed    2.32 sec
      Start  4: camera_if
 4/15 Test  #4: camera_if ........................   Passed    2.37 sec
      Start  5: cmd_ingest
 5/15 Test  #5: cmd_ingest .......................   Passed    2.29 sec
      Start  6: hs
 6/15 Test  #6: hs ...............................   Passed    2.39 sec
      Start  7: mapper
 7/15 Test  #7: mapper ...........................   Passed    2.32 sec
      Start  8: moonranger_msgs
 8/15 Test  #8: moonranger_msgs ..................   Passed    2.25 sec
      Start  9: planner
 9/15 Test  #9: planner ..........................   Passed    2.27 sec
      Start 10: pose_estimator
10/15 Test #10: pose_estimator ...................   Passed    2.20 sec
      Start 11: sb_transport_lib
11/15 Test #11: sb_transport_lib .................   Passed    2.22 sec
      Start 12: sch
12/15 Test #12: sch ..............................   Passed    2.22 sec
      Start 13: stereo_reconstructor
13/15 Test #13: stereo_reconstructor .............   Passed    2.28 sec
      Start 14: tlm_output
14/15 Test #14: tlm_output .......................   Passed    2.24 sec
      Start 15: vehicle_controller
15/15 Test #15: vehicle_controller ...............   Passed    2.32 sec

100% tests passed, 0 tests failed out of 15

Total Test time (real) =  34.34 sec
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [ ] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [x] New and existing unit tests pass locally with my changes
- [x] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- [ ] For new added message types, there is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader.

### CODE REVIEW

File: fsw/public_inc/stereo_reconstructor_msgs.h [hunk 1]
```diff
@@ -1,38 +1,51 @@
...
 
+/*************************************************************************/
+/*** STEREO App command codes*/
+#define STEREO_NOOP_CC 0
+#define STEREO_RESET_COUNTERS_CC 1
+
+/*************************************************************************/
+/*** Msg IDs*/
+
+#define STEREO_CMD_MID 0x18A0
```
Comment (kangkelvin) [inline fsw/public_inc/stereo_reconstructor_msgs.h:22]: This should be in the `moonranger_msgids.h`

**Review by kangkelvin — CHANGES_REQUESTED** [review]

File: fsw/public_inc/stereo_reconstructor_msgs.h (hunk 1 above)
Comment (WilliamLeeMoor) [reply fsw/public_inc/stereo_reconstructor_msgs.h]: Fixed, thank you!

**Review by tenzincrouch — CHANGES_REQUESTED** [review]
I left a couple of comments to address. Other than that this PR looks good to me.

File: fsw/public_inc/moonranger_msgids.h [hunk 2]
```diff
@@ -74,6 +74,9 @@
  * @note Telemetry message IDs in this section should fit within
  * 0x09C0-0x09FF inclusive.
  */
+#define STEREO_CMD_MID 0x18A0
```
Comment (kangkelvin) [inline fsw/public_inc/moonranger_msgids.h:77]: Please follow the pre-allocated msg ids addresses.

```
/**
 * Stereo Reconstructor Message IDs
 * @note Command message IDs in this section should fit within
 * 0x19C0-0x19FF inclusive.
 * @note Telemetry message IDs in this section should fit within
 * 0x09C0-0x09FF inclusive.
 */
```

**Review by kangkelvin — CHANGES_REQUESTED** [review]

**Review by tenzincrouch — APPROVED** [review]
LGTM

**Pull request merged**